internal/redis/redis_test.go

```diff
@@ -119,135 +119,161 @@ func TestICC(t *testing.T) {
 
 		if err != nil {
 			t.Fatalf("receiveApplause returned unexpected error: %v", err)
 		}
 
-		if applause != 1 {
+		if len(applause) != 1 {
 			t.Errorf("receiveApplause returned %d, expected 1", applause)
 		}
 	})
 
 	t.Run("Receive applause for one user", func(t *testing.T) {
 		defer r.ApplauseCleanOld(1000)
 
-		if err := r.ApplausePublish(1, 10); err != nil {
+		if err := r.ApplausePublish(1, 1, 10); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		applause, err := r.ApplauseReceive(10)
+		applause, err := r.ApplauseSince(10)
 
 		if err != nil {
 			t.Fatalf("receiveApplause returned unexpected error: %v", err)
 		}
 
-		if applause != 1 {
+		if applause[1] != 1 {
 			t.Errorf("receiveApplause returned %d, expected 1", applause)
 		}
 	})
 
 	t.Run("Receive applause for one user twice", func(t *testing.T) {
 		defer r.ApplauseCleanOld(1000)
 
-		if err := r.ApplausePublish(1, 10); err != nil {
+		if err := r.ApplausePublish(1, 1, 10); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		if err := r.ApplausePublish(1, 11); err != nil {
+		if err := r.ApplausePublish(1, 1, 11); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		applause, err := r.ApplauseReceive(10)
+		applause, err := r.ApplauseSince(10)
 
 		if err != nil {
 			t.Fatalf("receiveApplause returned unexpected error: %v", err)
 		}
 
-		if applause != 1 {
+		if applause[1] != 1 {
 			t.Errorf("receiveApplause returned %d, expected 1", applause)
 		}
 	})
 
 	t.Run("Receive applause for one user to old", func(t *testing.T) {
 		defer r.ApplauseCleanOld(1000)
 
-		if err := r.ApplausePublish(1, 9); err != nil {
+		if err := r.ApplausePublish(1, 1, 9); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		applause, err := r.ApplauseReceive(10)
+		applause, err := r.ApplauseSince(10)
 
 		if err != nil {
 			t.Fatalf("receiveApplause returned unexpected error: %v", err)
 		}
 
-		if applause != 0 {
+		if applause[1] != 0 {
 			t.Errorf("receiveApplause returned %d, expected 0", applause)
 		}
 	})
 
 	t.Run("Receive applause for two users", func(t *testing.T) {
 		defer r.ApplauseCleanOld(1000)
 
-		if err := r.ApplausePublish(1, 10); err != nil {
+		if err := r.ApplausePublish(1, 1, 10); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		if err := r.ApplausePublish(2, 10); err != nil {
+		if err := r.ApplausePublish(1, 2, 10); err != nil {
 			t.Fatalf("sending applause: %v", err)
 		}
 
-		applause, err := r.ApplauseReceive(10)
+		applause, err := r.ApplauseSince(10)
 
 		if err != nil {
 			t.Fatalf("receiveApplause returned unexpected error: %v", err)
 		}
 
-		if applause != 2 {
+		if applause[1] != 2 {
+			t.Errorf("receiveApplause returned %d, expected 2", applause)
+		}
+	})
+
+	t.Run("Receive applause for one user in two meetings", func(t *testing.T) {
+		defer r.ApplauseCleanOld(1000)
+
+		if err := r.ApplausePublish(1, 1, 10); err != nil {
+			t.Fatalf("sending applause: %v", err)
```

Comment:
Which `t` is here used? The first or the second one?

Reply:
The variable defined in line 255 (as your IDE can tell you). In go you can overwrite a variable.